python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -618,7 +613,6 @@ def ptp(self) -> dict | None:
             priority2 = self.shared_utils.id % 256
         default_auto_clock_identity = default(
             get(self._hostvars, "ptp_settings.auto_clock_identity"),
-            get(self._hostvars, "ptp.auto_clock_identity"),
             True,  # noqa: FBT003
         )
```

Comment:
Please change this to
`default_auto_clock_identity = get(self._hostvars, "ptp_settings.auto_clock_identity", default=True)`

Reply:
Done.